cmd/arc_summary/Makefile.am

```diff
@@ -1 +1,2 @@
 dist_bin_SCRIPTS = arc_summary.py
+dist_bin_SCRIPTS = arc_summary3.py
```

Comment:
Since `__init__.py` needs to be included in `make dist` you'll need to add and `EXTRA_DIST` line to this Makefile.am.

```diff
+EXTRA_DIST = __init__.py
```

Can we include a comment somewhere, either in  `__init__.py` or the `Makefile.am` describing why this empty file is needed.

Reply:
Sorry, hadn't thought of adding file to `test/runfiles/linux.run`. Still a bit hazy on the structure of the test suit ... 

As for __init__.py, let's let this run through the test suite again without it -- I think we might not need it after all, but because I'm having trouble duplicating the suite here, I wasn't sure. Thanks again for your help!